src/doc/yaml/cms.yaml

```diff
@@ -0,0 +1,565 @@
+
+# (c) Secure Identity Alliance
+
+openapi: 3.0.0
+info:
+  description: OSIA CMS Interface
+  version: 1.0.0
+  title: OSIA Credential Management System Interface
+  license:
+    name: SIA
+    url: "https://raw.githubusercontent.com/SecureIdentityAlliance/osia/master/LICENSE"
+tags:
+  - name: Credential Request
+servers:
+  - url: http://cms.com/
+  - url: https://cms.com/
+paths:
+
+  /v1/credentialRequest/person/{personId}:
+    post:
+      tags:
+        - Credential Request
+      summary: Create a request for a credential
+      operationId: createCredentialRequest
+      parameters:
+        - name: personId
+          in: path
+          description: the id of the person to create the credential request for
+          required: true
+          schema:
+            type: string
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CredentialRequest'
```

Comment:
I think the credential request should also include information regarding the credential build process: priority, delivery information, etc.)

Reply:
Updated following the last call.  Credential request now contains the person data instead of it being pulled from another system.